tools/react-app-template/LICENSE

```diff
@@ -0,0 +1,7 @@
+Copyright (c) 2020-2022 GoodData Corporation
```

Comment:
I think you will have to connect it to the same mechanism in place for sdk-ui-charts and others, because this package will not be MIT licence most likely (for the same reasons as sdk-ui-ext for example).

Reply:
Discussed with compliance and it's OK to leave the copyright headers in here. We'll skip the license altogether (just use `private: true` in `package.json`, as we have in the old template).